physics/cu_gf_driver_post.F90

```diff
@@ -15,28 +15,36 @@ module cu_gf_driver_post
 !> \section arg_table_cu_gf_driver_post_run Argument Table
 !! \htmlinclude cu_gf_driver_post_run.html
 !!
-   subroutine cu_gf_driver_post_run (im, t, q, prevst, prevsq, cactiv, cactiv_m, conv_act, conv_act_m, errmsg, errflg)
+   subroutine cu_gf_driver_post_run (im, km, t, q, prevst, prevsq, cactiv, cactiv_m, conv_act, conv_act_m,dt, garea, raincv, maxupmf, refl_10cm, errmsg, errflg)
 
       use machine, only: kind_phys
 
       implicit none
 
       ! Interface variables
-      integer,          intent(in)  :: im
+      integer,          intent(in)  :: im, km
       real(kind_phys),  intent(in)  :: t(:,:)
       real(kind_phys),  intent(in)  :: q(:,:)
+      real(kind_phys), dimension(:),intent(in) :: garea
       real(kind_phys),  intent(out) :: prevst(:,:)
       real(kind_phys),  intent(out) :: prevsq(:,:)
       integer,          intent(in)  :: cactiv(:)
       integer,          intent(in)  :: cactiv_m(:)
       real(kind_phys),  intent(out) :: conv_act(:)
       real(kind_phys),  intent(out) :: conv_act_m(:)
+      ! for Radar reflectivity
+      real(kind_phys),  intent(in)  :: dt
+      real(kind_phys),  intent(in)  :: raincv(:), maxupmf(:)
+      real(kind_phys),  intent(inout) :: refl_10cm(:,:)
       character(len=*), intent(out) :: errmsg
 !$acc declare copyin(t,q,cactiv,cactiv_m) copyout(prevst,prevsq,conv_act,conv_act_m)
       integer, intent(out)          :: errflg
 
       ! Local variables
-      integer :: i
+      real, parameter :: dbzmin=-10.0      ! dcd
```

Comment:
These should be real(kind_phys).

Reply:
@dustinswales  Thanks for pointing this out, and it will be updated.